src/components/Post/Comment/CommentsContainer.tsx

```
@@ -187,24 +196,58 @@ const CommentsContainer: FC<ICommentsContainerProps> = (props) => {
+			const { data, error } = await nextApiClientFetch<IReportSummaryResponse>('/api/v1/ai-summary/reportAISummary', {
+				postIndex,
+				postType
+			});
+
+			if (error || !data) {
+				console.log('Error While reporting AI summary data', error);
+				setReportingAISummary(false);
+				return;
+			}
+
+			if (data && data?.isAlreadyReported) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)